frontend/src/v4/modules/userManagement/userManagement.sagas.ts

```diff
@@ -282,6 +286,13 @@ export function* fetchProject({ project }) {
 
 export function* updateProjectPermissions({ permissions }) {
 	try {
+		const shouldUpdate = yield updatePermissionsOrTriggerModal({
```

Comment:
it seems that updating the permissions fails
![failed](https://github.com/user-attachments/assets/99bd4cda-7534-4b73-b9db-d8084f64e8f1)